src/tools/ceph_osdomap_tool.cc

```diff
@@ -180,6 +180,12 @@ int main(int argc, char **argv) {
     }
     for (auto i : headers)
       std::cout << i << std::endl;
+  } else if (cmd == "corrupt") {
```

Comment:
I haven't added testing of the "corrupt" command of ceph-osdomap-tool.  It doesn't mean anything after my last commit because we no longer use complete mappings.